CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 0.14.0 / 2015-06-01
+* [CHANGE] Configuration format changed and switched to YAML.
```

Comment:
Let's add a link to the migration tool here (and synchronize with pushing the new docs).